src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  position: relative;
+  height: 60px;
```

Comment:
it's better not hardcode height or elements with content inside. Use padding for your links